tests/mpi/collective_02_dealii_vector.mpirun=10.output

```diff
@@ -0,0 +1,2 @@
+
+DEAL:mpi::10 20
```

Comment:
I think it's enough to just run this test with 1 and 4 processors. No need to test too many variants on that example.